tests/EdFi.Admin.DataAccess.IntegrationTests/Models/UserContextMappingTests.cs

```diff
@@ -18,6 +19,8 @@ namespace EdFi.Ods.Admin.DataAccess.IntegrationTests.Models
 {
     public class UserContextMappingTests
     {
+        private static string connectionString = ConfigurationManager.ConnectionStrings["EdFi_Admin"]
```

Comment:
ConfigurationManager.ConnnectionStrings is not valid in net core. you need to use the Configuration.GetConnectionStrings() as we are moving to appsettings.json

Reply:
Also the naming convention is wrong for fields.

Reply:
The more I think about this, you should probably use IConnectionStringProvider instead